schemas/hub/definitions.json

```diff
@@ -14,58 +14,44 @@
           "$ref": "https://eq.ons.gov.uk/common_definitions.json#/non_empty_string"
         }
       },
-      "complete": {
+      "submission": {
         "type": "object",
         "properties": {
+          "button": {
+            "$ref": "https://eq.ons.gov.uk/common_definitions.json#/non_empty_string",
+            "description": "The text that appears on the submit button, when the questionnaire is ready to be submitted"
+          },
+          "guidance": {
+            "$ref": "https://eq.ons.gov.uk/common_definitions.json#/non_empty_string",
+            "description": "The text guidance that appears, when the questionnaire is ready to be submitted"
+          },
           "title": {
             "$ref": "https://eq.ons.gov.uk/common_definitions.json#/non_empty_string",
             "description": "The title that will appear on the hub, when the questionnaire is complete."
           },
-          "guidance": {
-            "$ref": "https://eq.ons.gov.uk/hub/definitions.json#/guidance"
+          "warning": {
+            "$ref": "https://eq.ons.gov.uk/common_definitions.json#/non_empty_string",
+            "description": "The warning that will appear on the hub, when the questionnaire is complete."
```

Comment:
The descriptions should be consistent and all use "when the questionnaire is complete" rather than "when the questionnaire is ready to be submitted".